include/kodlab_mjbots_sdk/pid.h

```diff
@@ -0,0 +1,180 @@
+/**
+ * @file pid.h
+ * @author Chandravaran Kunjeti (kunjeti@seas.upenn.edu)
+ * @brief Class implementation of PID
+ * @date 9/30/2022
+ *
+ * @copyright 2022 The Trustees of the University of Pennsylvania.
+ * All rights reserved.
+ *
+ */
+
+#pragma once
+
+#include <math.h>
+
+#include <vector>
+
+#define HIGH_SATURATION_INDEX_ 0
+#define LOW_SATURATION_INDEX_ 0
+
+namespace kodlab {
+
```

Comment:
Once again, namespace contents [should **NOT** be indented](https://drake.mit.edu/styleguide/cppguide.html#Namespace_Formatting).